app/models/topic.rb

```diff
@@ -28,6 +29,8 @@ class Topic < ApplicationRecord
   STATES = %i[active archived].freeze
   CONTENT_TYPES = %w[image/jpeg image/png image/svg+xml image/webp image/avif image/gif video/mp4 application/pdf audio/mpeg].freeze
 
+  default_scope { where(shadow_copy: false) }
```

Comment:
Shadow copy topics will be collected in our DB, but won't be visible.
We can make a new job to clean them from time to time.